### DISCUSSION TITLE
Support more dictionary formats

### DISCUSSION THREAD

**Discussion by GrimPixel** [xiaoyifang/goldendict-ng#1100] [Announcements]
**Is your feature request related to a problem? Please describe.**
There are some formats, like CC-CEDICT, to be supported. The most simple format TSV is not supported yet.

**Describe the solution you'd like**
Support as many formats mentioned in https://github.com/ilius/pyglossary as possible.

**Comment by xiaoyifang** [comment]
For now ,you can use pyglossary to convert other format into a format which goldendict supports.

btw, PRs support other format is welcome

**Comment by xiaoyifang** [comment]
as for CC-CEDICT ,some community guy have converted it into mdx format.
